### ISSUE TITLE
Input: Form field errors are not associated with their corresponding field

### ISSUE THREAD

**Issue by sjasaniel** [infor-design/enterprise#4830]
**Describe the bug**
Error messages are not associated with inputs. Screen reader users may not be able to identify the relationships between inputs and error messages.

**Expected behavior**
Developers should provide a programmatic association between inputs and error messages. This can be done using the aria-described attribute on the input, referencing the ID of the error message.
Note: adding aria-invalid="true" to inputs with errors will further aid screen reader users in completing forms and correcting their errors.

[Compliant Code Sample]
```
<label for="nationality">
<input aria-describedby="nationality-error" aria-invalid="true" type="text" id="nationality">
<div id="nationality-error">Field Nationality is required</div>
```

**Version**
IDS Enterprise Components - v4.36.2

**Platform**
Infor Application/Team Name: Landmark Client
Device: (if applicable) : N\A
OS Version: Windows 10
Browser Name: Chrome
Browser Version: 88.0.4324.146 (Official Build) (64-bit)

**Additional context**
https://jira.lawson.com/browse/LMCLIENT-34281

**Comment by tmcconechy** [comment]
How are you guys attaching these errors - i would make sure you not doing something custom. And that it works like either this:

https://master-enterprise.demo.design.infor.com/components/validation/example-index.html
or this
https://master-enterprise.demo.design.infor.com/components/validation/test-message-types.html (i think its this)

**Comment by CindyMercadoReyes** [comment]
This issue is now resolved.
